mlflow/metrics/utils/model_utils.py

```diff
@@ -0,0 +1,73 @@
+import asyncio
+import os
+import re
+
+from mlflow.exceptions import MlflowException
+from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE
+
+ROUTE_TYPE = "llm/v1/completions"
+
+
+# TODO: improve this name
+def score_model_on_payload(model_uri, payload):
+    """Call the model identified by the given uri with the given payload."""
+
+    prefix, suffix = _parse_model_uri(model_uri)
+
+    if prefix == "openai":
+        return _call_openai_api(suffix, payload)
+    elif prefix == "gateway":
+        return _call_gateway_api(suffix, payload)
+    elif prefix == "model":
+        # TODO: replace with call_pyfunc_model_api
```

Comment:
I think this is the function for other str: `_load_model_or_server` imported from:

`from mlflow.pyfunc import  _load_model_or_server`

Also we shouldn't do `model:` only, there are cases where `_load_model_or_server` I think works for `runs:` too. So lets add an else case like a catch all or else throw

Reply:
@sunishsheth2009 @prithvikannan Can we start without supporting `runs:/` / `models:/`? I I recall a discussion about how we need environment restoration for the LLM judge, which complicates the API impl. IMO would be best to punt on that for now

Reply:
Ack. We can scope this to `openai:/` and `gateway:/` for now.

Reply:
I think we do plan to use `_EnvManager.LOCAL` so users can at least use Local logged model if they don't use databricks or gateway for example. 
Since this would force users to use gateway(what if they are not enabled to gateway). So thought lets do one option at least. Thoughts?